MediaBrowser.Providers/Manager/ProviderManager.cs

```diff
@@ -807,13 +807,11 @@ private bool IsSaverEnabledForItem(IMetadataSaver saver, BaseItem item, LibraryO
             where TLookupType : ItemLookupInfo
         {
             var results = await provider.GetSearchResults(searchInfo, cancellationToken).ConfigureAwait(false);
-
-            foreach (var item in results)
```

Comment:
Here we have a bug. If `provider.GetSearchResults` implementation returns lazy enumeration (like `MusicBrainzArtistProvider` for example), we lose `SearchProviderName`

Reply:
Why not remove the method? It's a private method used in only one place.